external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -588,3 +602,39 @@ def integrate_stepwise(self, ds: xr.Dataset) -> xr.Dataset:
         return xr.concat(state_out_list, dim="time").transpose(
             self.sample_dim_name, "time", "z"
         )
+
+    @classmethod
+    def load(cls, path: str) -> "StepwiseModel":
+        """Load a serialized model from a directory."""
+        with get_dir(path) as path:
+            model_filename = os.path.join(path, cls._MODEL_FILENAME)
+            model = tf.keras.models.load_model(
+                model_filename, custom_objects=cls.custom_objects
+            )
+            with open(os.path.join(path, cls._CONFIG_FILENAME), "r") as f:
+                config = yaml.load(f, Loader=yaml.Loader)
+            obj = cls(
+                config["input_variables"],
+                config["output_variables"],
+                config.get("output_metadata", None),
+                model,
+                config["sample_dim_name"],
+            )
+            return obj
+
+    def dump(self, path: str) -> None:
+        with put_dir(path) as path:
+            if self.model is not None:
+                model_filename = os.path.join(path, self._MODEL_FILENAME)
+                self.model.save(model_filename)
+            with open(os.path.join(path, self._CONFIG_FILENAME), "w") as f:
+                f.write(
+                    yaml.dump(
+                        {
+                            "input_variables": self.input_variables,
+                            "output_variables": self.output_variables,
+                            "output_metadata": self._output_metadata,
+                            "sample_dim_name": self.sample_dim_name,
+                        }
+                    )
+                )
```

Comment:
Why is this appearing as added code? Did you add it? If so then why, it seems unrelated?

Reply:
OK.